src/main/resources/db/changelog/ddl/0013_initial_retention_data_removed_column.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="matt.wills@valtech.com" id="0013_EIP1-4053_alter_certificate - add initial_retention_data_removed column" context="ddl">
+
+        <addColumn tableName="certificate">
+            <column name="initial_retention_data_removed" type="boolean" value="false"
+                    remarks="Indicates if initial retention period data has been removed from a certificate">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <dropIndex tableName="certificate" indexName="certificate_initial_retention_removal_date_idx"/>
+
+        <createIndex tableName="certificate" indexName="certificate_initial_retention_removal_idx">
+            <column name="source_type"/>
+            <column name="initial_retention_data_removed"/>
+            <column name="initial_retention_removal_date"/>
+        </createIndex>
+
+        <rollback>
```

Comment:
Does `certificate_initial_retention_removal_idx` need dropping before column can be dropped?

Reply:
Good point. According to the mysql docs it looks like the column is just removed from the index, so it would cause issues when the changeset is tried on subsequent occasions.

Reply:
Done.